app/uk/gov/hmrc/economiccrimelevyregistration/models/RegistrationType.scala

```diff
@@ -25,29 +25,33 @@ object RegistrationType {
 
   case object Initial extends RegistrationType("Initial")
   case object Amendment extends RegistrationType("Amendment")
+  case object Dereg extends RegistrationType("Dereg")
 
-  lazy val values: Set[RegistrationType] = Set(Initial, Amendment)
+  lazy val values: Set[RegistrationType] = Set(Initial, Amendment, Dereg)
 
   implicit val format: Format[RegistrationType] = new Format[RegistrationType] {
     override def reads(json: JsValue): JsResult[RegistrationType] = json.validate[String] match {
       case JsSuccess(value, _) =>
         value match {
           case "Initial"   => JsSuccess(Initial)
           case "Amendment" => JsSuccess(Amendment)
+          case "Dereg"     => JsSuccess(Dereg)
         }
       case e: JsError          => e
     }
 
     override def writes(o: RegistrationType): JsValue = o match {
       case Initial   => JsString("Initial")
       case Amendment => JsString("Amendment")
+      case Dereg     => JsString("Dereg")
     }
   }
 
   implicit val jsLiteral: JavascriptLiteral[RegistrationType] = new JavascriptLiteral[RegistrationType] {
     override def to(value: RegistrationType): String = value match {
       case Initial   => "Initial"
       case Amendment => "Amendment"
+      case Dereg     => "Dereg"
```

Comment:
See other comment.

Reply:
This one still says Dereg @paddynski-opencast

Reply:
Corrected.